object.c

```diff
@@ -4194,23 +4194,16 @@ f_sprintf(int c, const VALUE *v, VALUE _)
  *
  *  These are the methods defined for \BasicObject:
  *
- *  - ::new:: Returns a new \BasicObject instance.
- *  - #!:: Returns the boolean negation of +self+: +true+ or +false+.
- *  - #!=:: Returns whether +self+ and the given object are _not_ equal.
- *  - #==:: Returns whether +self+ and the given object are equivalent.
- *  - {__id__}[#method-i-__id__]:: Returns the integer object identifier for +self+.
- *  - {__send__}[#method-i-__send__]:: Calls the method identified by the given symbol.
- *  - #equal?:: Returns whether +self+ and the given object are the same object.
- *  - #instance_eval:: Evaluates the given string or block in the context of +self+.
- *  - #instance_exec:: Executes the given block in the context of +self+,
- *                     passing the given arguments.
- *  - #method_missing:: Method called when an undefined method is called on +self+.
- *  - #singleton_method_added:: Method called when a singleton method
- *                              is added to +self+.
- *  - #singleton_method_removed:: Method called when a singleton method
- *                                is added removed from +self+.
- *  - #singleton_method_undefined:: Method called when a singleton method
- *                                  is undefined in +self+.
+ *  - ::new: Returns a new \BasicObject instance.
+ *  - {!}[#method-i-21]: Returns the boolean negation of +self+: +true+ or +false+.
+ *  - {!=}[#method-i-21-3D]: Returns whether +self+ and the given object are _not_ equal.
```

Comment:
```suggestion
 *  - #!: Returns the boolean negation of +self+: +true+ or +false+.
 *  - #!=: Returns whether +self+ and the given object are _not_ equal.
```

Reply:
The suggested revisions above do not become links on my system:
- $ ruby --version
  ruby 3.1.0p0 (2021-12-25 revision fb4df44d16) [x64-mingw-ucrt]
- $ rdoc --version
  6.4.0

Is this fixed in later versions?

There are many more like this in the What's Here sections.  When they are no longer needed, I'll be happy to do a sweep through to clean them up.

Reply:
It is fixed at https://github.com/ruby/rdoc/pull/865 and has been merged to the ruby master, but rdoc is not released yet.